src/execute/rmd.ts

```diff
@@ -279,7 +279,8 @@ function withinActiveRenv() {
 }
 
 async function printCallRDiagnostics() {
-  const caps = await knitrCapabilities();
+  const rBin = await rBinaryPath("Rscript");
+  const caps = await knitrCapabilities(rBin);
```

Comment:
This should be improved to do the same checking as in `checkKnitrInstallation()`. However, I am doing the minimum here so that it does not have impact for now. I'll update this in another PR as discussed with @cscheid